cherrypy/__init__.py

```diff
@@ -57,7 +57,7 @@
 """
 
 try:
-    import pkg_resources
+    import importlib
 except ImportError:
     pass
```

Comment:
Probably you'll also need to fallback to `import importlib_metadata` unless support for Python 3.7 has been dropped.

Reply:
fix in new patch